### PR TITLE
bump action versions

### PR DESCRIPTION
**Pull request by larsrh**
Should hopefully address the failure in #430

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -24,10 +24,3 @@ jobs:
           restore-keys: ${{ runner.os }}-sbt-
       - name: Publish
         run: csbt ci-release
-      - name: Publish 0.6
```
Comment (larsrh) [inline .github/workflows/release.yml:27]: I forgot to delete this in #426

**Comment by cquiroz** [comment]
Thanks, should we include a dependably config to keep actions up to date?

**Review by cquiroz — APPROVED** [review]
LGTM

**Comment by larsrh** [comment]
> Thanks, should we include a dependably config to keep actions up to date?

Up for you to decide, but I'm happy to set it up.

**Pull request merged**